### DISCUSSION TITLE
NOOB Question: How to modify YAML

### DISCUSSION THREAD

**Discussion by sanderbaks** [davidrapan/ha-solarman#141] [Q&A]
Hello!

So I am trying to set up my solar panels. There for I need to modify some YAML parameter files and see the reults of those changes.

Now, I am unclear of the procedure I need to execute. I use Studio Code Server to see and modify files.
Do I make the changes and just "Reload YAML"? Or do I need to reload HA as a whole?

Thanks in advance!

Sander

**Comment by davidrapan** [comment]
Hi @sanderbaks, if you are trying to "scan" over registers and so better and way quicker will be to do it using services which are exposed through the integration. 😉 

But deleting the device and re-adding it back should be sufficient after the changes.

**Comment by sanderbaks** [comment]
Hmm, I am not sure what you mean by "Using Services" ? I indeed want to scan through all the possible registers to find meaningfull matches.

**Reply by davidrapan** [reply]
In Developer Tools there is tab ACTIONS. There you can find services under Solarman domain and use that to read particular registers.

**Comment by sanderbaks** [comment]
WOW, I did not know about this :)
Then would I be looking for Holding or Input registers ?

**Reply by davidrapan** [reply]
Holding in most of the cases.
